Aptoide/aptoide-client-v8
# [hotfix/ASV-978] fix app coins card title

**What does this PR do?**
Center the AppCoins card title on all android versions.

**Database changed?**

    No

**Where should the reviewer start?**

- [ ] info_action_item.xml
- [ ] v21/info_action_item.xml

**How should this be manually tested?**

  Open on devices above and bellow 21, put the device in a language with a longer title translation and check if it is centered

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-978](https://aptoide.atlassian.net/browse/ASV-978)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
